scripts/install-chef-suse.sh

```diff
@@ -665,6 +665,16 @@ if [ ! -f /etc/rabbitmq/rabbitmq.config ] ; then
 EOF
 fi
 
+# Tell system-widde epmd to listen on right address (new rabbitmq-server
+# packages use this epmd service)
+mkdir -p /etc/systemd/system/epmd.socket.d
+cat << EOF > /etc/systemd/system/epmd.socket.d/port.conf
+[Socket]
+ListenStream=$IPv4_addr:4369
+FreeBind=true
```

Comment:
```
 This option, when enabled, allows sockets to be bound to a non-local, not configured IP address, and hence allows bindings to a particular IP address before it actually becomes available, making such an explicit dependency to the configured address redundant
```
whats the reason for this? Is the network not available by this time?

Reply:
On reboot, the socket units are started before network.target (and wicked), which means before we get the IP address assigned to any interface. So without this, things fail on reboot.